### 爬虫中所碰到的问题总结记录

#### Caused by ProxyError('Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response'))

实时更新请求头的 User-Agent   fake_useragent

[代理异常](https://zhuanlan.zhihu.com/p/105602367)

[代理Max retries exceeded with url](https://github.com/zyl-fun/blog/blob/master/%E4%BB%A3%E7%90%86Max%20retries%20exceeded%20with%20url.md)

![img](%E5%9B%BE%E7%89%87/d398-hvntnkp6748871.jpg)